edns.go

```diff
@@ -159,6 +159,8 @@ type EDNS0 interface {
 	unpack([]byte) error
 	// String returns the string representation of the option.
 	String() string
+	// Copy return deep copy of this EDNS0 information
```

Comment:
`// Copy returns a deep-copy of the option.`

Reply:
ok.